src/redux/reducers/samples/samplesFileUpdate.js

```diff
@@ -20,7 +20,11 @@ const samplesFileUpdate = (state, action) => {
 
   newFile.lastModified = lastModified;
 
-  const newFileNames = Array.from(new Set([...state[sampleUuid].fileNames, fileName]));
```

Comment:
The main reason we were using a set was to ensure that there is no duplicate name.